tests/test_backend.py

```diff
@@ -132,6 +132,28 @@ def test_send_mail(self):
         self.assertEqual(mail['to'], 'to@example.com')
         self.assertEqual(mail.get_payload(), 'body')
 
+    def test_send_mail_when_blacklisted(self):
```

Comment:
Should add a test verifying not sent when email is in blacklist, and one with case-sensitive logic too to test the new flag.

Reply:
That testcase tests both cases (trying to send an email to a not-blacklisted-email and to a blacklisted-email). Maybe I'm not understanding you? Do yo want me to split the testcase in two? 🤔

Reply:
I see the second assertion when an email is already blacklisted.

Maybe it makes sense to also test sending to two email addresses - one blacklisted and one not, to make sure the email message still gets sent but only to the allowed address?

Reply:
Ah, yes, good point! Let me push another testcase!